### DISCUSSION TITLE
How to disaply in-place only my string (without progress bar, etc.)?

### DISCUSSION THREAD

**Discussion by simon-liebehenschel** [tqdm/tqdm#1317] [Q&A]
I do NOT need:
- progress bar
- 'tqdm' calculations
- nothing that is displayed by default!

I need:
- Display dynamically in-place my custom string

**Comment by nishnash54** [comment]
You can make use of the `bar_format` parameter to format the information displayed in the tqdm status bar. It also has a description field which can be set and updated using the `set_description()` method.
```py
import time
import random
from tqdm import tqdm

desc = tqdm(range(10), bar_format="{desc}")
for i in desc:
    x = random.random()
    if x > 0.75:
        desc.set_description(f"High: {x} > 0.75")
    else:
        desc.set_description(f"Low: {x} <= 0.75")
    time.sleep(1)
```

**Reply by zazaplugofficial** [reply]
https://github.com/tqdm/tqdm/discussions/1317?sort=top#discussioncomment-3078996
